[2022-08-18 17:40:04,248] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: elastic_desk.popula_elastic_finalizados scheduled__2022-08-18T17:35:00+00:00 [queued]>
[2022-08-18 17:40:04,408] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: elastic_desk.popula_elastic_finalizados scheduled__2022-08-18T17:35:00+00:00 [queued]>
[2022-08-18 17:40:04,408] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-18 17:40:04,408] {taskinstance.py:1377} INFO - Starting attempt 1 of 1
[2022-08-18 17:40:04,408] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-18 17:40:04,491] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): popula_elastic_finalizados> on 2022-08-18 17:35:00+00:00
[2022-08-18 17:40:04,508] {standard_task_runner.py:52} INFO - Started process 8726 to run task
[2022-08-18 17:40:04,514] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'elastic_desk', 'popula_elastic_finalizados', 'scheduled__2022-08-18T17:35:00+00:00', '--job-id', '50', '--raw', '--subdir', 'DAGS_FOLDER/etl_elastic_desk.py', '--cfg-path', '/tmp/tmpzj9l6thu', '--error-file', '/tmp/tmpqbfl_lwo']
[2022-08-18 17:40:04,527] {standard_task_runner.py:80} INFO - Job 50: Subtask popula_elastic_finalizados
[2022-08-18 17:40:04,756] {task_command.py:371} INFO - Running <TaskInstance: elastic_desk.popula_elastic_finalizados scheduled__2022-08-18T17:35:00+00:00 [running]> on host 1e1070d9e5ae
[2022-08-18 17:40:05,034] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=clecio.antao@gmail.com
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=elastic_desk
AIRFLOW_CTX_TASK_ID=popula_elastic_finalizados
AIRFLOW_CTX_EXECUTION_DATE=2022-08-18T17:35:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-08-18T17:35:00+00:00
[2022-08-18 17:40:05,043] {base.py:271} INFO - GET http://192.168.2.100:9200/ [status:200 request:0.006s]
[2022-08-18 17:40:09,466] {base.py:271} INFO - HEAD http://192.168.2.100:9200/_template/desk_finalizados [status:200 request:0.002s]
[2022-08-18 17:40:10,028] {logging_mixin.py:115} WARNING - /home/***/.local/lib/python3.7/site-packages/es_pandas/es_pandas.py:58 RuntimeWarning: invalid value encountered in double_scalars
[2022-08-18 17:40:10,296] {base.py:271} INFO - POST http://192.168.2.100:9200/_bulk [status:200 request:0.137s]
[2022-08-18 17:40:10,513] {base.py:271} INFO - POST http://192.168.2.100:9200/_bulk [status:200 request:0.168s]
[2022-08-18 17:40:10,662] {base.py:271} INFO - POST http://192.168.2.100:9200/_bulk [status:200 request:0.178s]
[2022-08-18 17:40:10,745] {base.py:271} INFO - POST http://192.168.2.100:9200/_bulk [status:200 request:0.089s]
[2022-08-18 17:40:10,839] {base.py:271} INFO - POST http://192.168.2.100:9200/_bulk [status:200 request:0.077s]
[2022-08-18 17:40:11,069] {base.py:271} INFO - POST http://192.168.2.100:9200/_bulk [status:200 request:0.142s]
[2022-08-18 17:40:11,221] {base.py:271} INFO - POST http://192.168.2.100:9200/_bulk [status:200 request:0.121s]
[2022-08-18 17:40:11,279] {base.py:271} INFO - POST http://192.168.2.100:9200/_bulk [status:200 request:0.070s]
[2022-08-18 17:40:11,344] {base.py:271} INFO - POST http://192.168.2.100:9200/_bulk [status:200 request:0.051s]
[2022-08-18 17:40:11,375] {base.py:271} INFO - POST http://192.168.2.100:9200/_bulk [status:200 request:0.026s]
[2022-08-18 17:40:11,437] {logging_mixin.py:115} INFO - 0       18-08-2022
1       18-08-2022
2       18-08-2022
3       18-08-2022
4       18-08-2022
           ...    
4751    19-07-2022
4752    19-07-2022
4753    19-07-2022
4754    19-07-2022
4755    19-07-2022
Name: DataFinalizacao, Length: 4756, dtype: object
[2022-08-18 17:40:11,440] {python.py:173} INFO - Done. Returned value was: None
[2022-08-18 17:40:11,478] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=elastic_desk, task_id=popula_elastic_finalizados, execution_date=20220818T173500, start_date=20220818T174004, end_date=20220818T174011
[2022-08-18 17:40:11,558] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-18 17:40:11,602] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
